src/main/java/frc/robot/vision/AprilTagCamera.java

```diff
@@ -316,7 +316,7 @@ private void updateEstimatedGlobalPose() {
         if (tag.fiducialId == 12) {
           distTo12 = UtilFunctions.getDistance(tag.bestCameraToTarget);
         }
-        if (tag.fiducialId == 18) {
+        if (tag.fiducialId == 7) {

```

Comment:
all the 18 to 7 changes here was because I was working on the red side of the field, so I was able to calculate distances on the same side of the field.  This should probably be cleaned up...